docker/devel:openQA:ci/base/Dockerfile

```diff
@@ -3,10 +3,10 @@
 FROM opensuse/leap:15.1
 
 # these are autoinst dependencies
-RUN zypper install -y autoconf automake gcc-c++ libtool pkgconfig\(opencv\) pkg-config perl\(Module::CPANfile\) pkgconfig\(fftw3\) pkgconfig\(libpng\) pkgconfig\(sndfile\) pkgconfig\(theoraenc\) make
+RUN zypper install -y os-autoinst-devel
```

Comment:
This is nice but not sure if it will work forever as the devel package is from the published package but then we try to build os-autoinst from git. I know, this all sucks :( well, I still think this is better, so leave it.

Reply:
I've already removed it. It would also be problematic because it would pull lots of Perl dependencies and I'm not sure whether it would interfere with the dependency tracking in `ci-packages.txt`.